nan.h

```diff
@@ -1654,12 +1661,12 @@ class Callback {
       Execute(progress);
   }
 
-  void SendProgress_(const char *data, size_t size) {
-    char *new_data = new char[size];
+  void SendProgress_(T *data, size_t size) {
+    T *new_data = new T[size];
     memcpy(new_data, data, size);
```

Comment:
Nevermind. I missed that it is supposed to be an array. I was only reading the e-mail notifications, missing out on the full context.

On June 30, 2016 8:11:35 PM GMT+03:00, Ben Noordhuis <notifications@github.com> wrote:
>> @@ -1654,12 +1661,12 @@ class Callback {
>>        Execute(progress);
>>    }
>>  
>> -  void SendProgress_(const char *data, size_t size) {
>> -    char *new_data = new char[size];
>> +  void SendProgress_(T *data, size_t size) {
>> +    T *new_data = new T[size];
>>      memcpy(new_data, data, size);
>
>I must confess I don't quite understand what you're getting at.
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/nodejs/nan/pull/588/files/85aa1f0be5361cd79a2c85f75cb6296188223c87#r69171812